Makefile

```diff
@@ -184,6 +207,26 @@ test-timers:
 test-timers-clean:
 	$(MAKE) --directory=tools clean
 
+test-v8:
+	# note: performs full test unless QUICKCHECK is specified
+	deps/v8/tools/run-tests.py --arch=$(V8_ARCH) --mode=$(BUILDTYPE_LOWER) $(V8_TEST_NO_I18N) \
+	  $(QUICKCHECK_ARG) --no-presubmit --shell-dir=$(PWD)/deps/v8/out/$(V8_ARCH).$(BUILDTYPE_LOWER) \
+	 $(TAP_V8)
```

Comment:
Indent seems a little off here and 10 lines down.  Also, if you can keep these lines <= 80 columns, that'd be awesome.